.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
 # code analysis
   test-static-code-analysis:
-    resource_class: medium+
+    resource_class: large
```

Comment:
Oh we really need to consider splitting the project, it is more and more obvious the code base in this repo is large enough already.